src/org/zaproxy/zap/extension/spider/SpiderAPI.java

```diff
@@ -289,15 +290,17 @@ private GenericScanner2 getSpiderScan(JSONObject params) throws ApiException {
 	 * The {@code scanIdCounter} is used to generate the ID of the started spider scan. The started {@code SpiderScan} is saved
 	 * in {@code spiderScans} for later access/control, accessible with the returned ID.
 	 * </p>
-	 * 
+	 *
 	 * @param url the url to start the spider scan
 	 * @param user the user to scan as, or null if the scan is done without the perspective of any user
+	 * @param maxChildren Max number of children to scan
+	 * @param recurse Whether or not to scan recursively
```

Comment:
I wonder if "recurse" is the best name for this option (the same for GUI).
The "scan recursively" might be misleading, the spider will still scan recursively even when it's false. The option just controls the pre-scan behaviour, from help:
> If you select 'recurse' then all of the nodes underneath the one selected will also be used to seed the spider.